tests/helpers.py

```diff
@@ -80,21 +80,21 @@ def _check_log_for_message(device, service, message, since=None):
             time.sleep(10)
             sleepsec += 10
             logger.info(
-                f"waiting for message '{message}' in {service} log, waited for: {sleepsec}"
+                f"waiting for message '{message}' in mender-authd log, waited for: {sleepsec}"
             )
 
         assert (
             sleepsec <= timeout
-        ), f"timeout for waiting for message '{message}' in {service} log"
+        ), f"timeout for waiting for message '{message}' in mender-authd log"
 
     @staticmethod
     def check_log_is_authenticated(device, since=None):
         Helpers._check_log_for_message(
-            device, "mender-updated", "No update available", since
+            device, "Successfully received new authorization data", since
```

Comment:
Lol, several messages were posted, then deleted again, as I followed this evolution in the commits! 😂

Reply:
Yeah, sorry for that. There was a bit of back and forth as I found issues in the tests.